core/domain/fs_services.py

```diff
@@ -458,5 +458,11 @@ def get_static_asset_url(filepath: str) -> str:
         return 'http://localhost:8181/assetsstatic/%s' % (
             filepath
         )
+    from core.domain import platform_parameter_list
```

Comment:
Perhaps add a comment here saying why the inline imports are needed. There is an existing TODO, I think, about removing inline imports -- perhaps you could add a link to that here, too.

Reply:
Added, let me know if anything is unclear.

Reply:
There are two other feconf constants dependent on oppia_project_id:

- `ENV_IS_OPPIA_ORG_PRODUCTION_SERVER`
- `OPPIA_SITE_URL`

Another approach here can be creating a util functions in feconf or a new lib `core/project_id_conf.py` to fetch these three params which limits the import to that lib instead of adding it in scripts and introducing multiple inline imports. WDYT?